tests/queries/0_stateless/03282_json_equal_comparison.reference

```diff
@@ -0,0 +1,16 @@
+{"a":0}	{"a":0}	1
+{"a":42}	{"a":42}	1
+{"a":42}	{"a":43}	0
+{"a":42,"b":"42"}	{"a":42,"b":"42"}	1
+{"a":42,"b":"42"}	{"a":42,"b":"43"}	0
+{"a":42,"b":"42","c":"42"}	{"a":42,"b":"42","d":"42"}	0
+{"a":42,"b":"42","c":"42","d":"42"}	{"a":42,"b":"42","c":"42","d":"42"}	1
+{"a":42,"b":"42","c":"42","d":"42"}	{"a":42,"b":"42","c":"42","d":"43"}	0
+{"a":42,"b":"42","c":"42","d":"42"}	{"a":42,"b":"42","c":"43","d":"42"}	0
+{"a":42,"b":"42","d":"42"}	{"a":42,"b":"42","d":"42"}	1
+{"a":42,"b":"42","d":"42"}	{"a":42,"b":"42","c":"42","d":"42"}	0
+{"a":42,"b":"42","c":"42"}	{"a":42,"b":"42","c":"42"}	1
+{"a":42,"b":"42","c":"42"}	{"a":42,"b":"42","c":"42","d":"42"}	0
+{"a":42,"b":"42","c":"42","d":"42","e":"42"}	{"a":42,"b":"42","c":"42","d":"42","e":"42"}	1
+{"a":42,"b":"42","c":"42","d":"42","e":"42"}	{"a":42,"b":"42","c":"42","d":"42","e":"42"}	0
```

Comment:
Do we already have some documentation about JSON type? If so, we could describe how comparisons are performed, e.g. about different types inside objects, as it's not the same as comparing all values.

Reply:
We have documentation: https://clickhouse.com/docs/en/sql-reference/data-types/newjson. I will add a section about comparison.